packages/css/src/plugins/atomicify-rules.tsx

```diff
@@ -171,24 +190,26 @@ const atomicifyAtRule = (node: AtRule, opts: AtomicifyOpts): AtRule => {
   node.each((childNode) => {
     switch (childNode.type) {
       case 'atrule':
-        children.push(atomicifyAtRule(childNode, atRuleOpts));
+        children.push(atomicifyAtRule(childNode, atRuleOpts, postcss));
         break;
 
       case 'rule':
-        atomicifyRule(childNode, atRuleOpts).forEach((rule) => {
+        atomicifyRule(childNode, atRuleOpts, postcss).forEach((rule) => {
           children.push(rule);
         });
         break;
 
       case 'decl':
-        children.push(atomicifyDecl(childNode, atRuleOpts));
+        children.push(atomicifyDecl(childNode, atRuleOpts, postcss));
         break;
 
       default:
         break;
     }
   });
 
+  newNode.nodes = children;
```

Comment:
Is this needed when it's initialised above?

Reply:
It wasn't working without reassignment, perhaps due to some change in how the clone behaviour deals with references.
I've updated this to push new node items directly to newNode.nodes, and this resolves the need for the reassignment